eq-author/src/components/Comments/CommentEditor/index.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { colors, focusStyle } from "constants/theme";
+
+import Button from "components/buttons/Button";
+import TextareaAutosize from "react-textarea-autosize";
+
+const Wrapper = styled.div`
+  margin-bottom: 1em;
+`;
+
+const TextAreaFixedSize = styled.textarea`
+  height: 94px;
+  width: 100%;
+  border: thin solid ${colors.grey};
+  resize: none;
+  font-size: 1em;
+  font-family: inherit;
+  padding: 0.5em;
+  margin-bottom: 0.5em;
+
+  &:focus {
+    ${focusStyle}
+    outline: none;
+  }
+`;
+
+const TextAreaGrowable = styled(TextareaAutosize)`
+  width: 100%;
+  border: thin solid ${colors.grey};
+  resize: none;
+  font-size: 1em;
+  font-family: inherit;
+  padding: 0.5em;
+  margin-bottom: 0.5em;
+
+  &:focus {
+    ${focusStyle}
+    outline: none;
+  }
+`;
```

Comment:
I think there's one css property difference between these. Could make it into a common css property thing